### PR TITLE
New module:  mmseqs/createtsv

### PR DESCRIPTION
**Pull request by Joon-Klaps**
## PR checklist

Create a summary tsv file of a query target and result MMseqs database

- [X] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by Joon-Klaps** [comment]
See the test config to specify the selection of the cluster database (based on the knowledge that this database has the suffix `_clu.dbtype`).

File: tests/modules/nf-core/mmseqs/createtsv/main.nf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+include { UNTAR as UNTAR_RESULT2 } from '../../../../../modules/nf-core/untar/main.nf'
+include { MMSEQS_CREATETSV       } from '../../../../../modules/nf-core/mmseqs/createtsv/main.nf'
+
+workflow test_mmseqs_createtsv {
+
+    input_file = file(params.test_data['sarscov2']['genome']['mmseqs_tar_gz'], checkIfExists: true)
+
+    ch_query = UNTAR_QUERY ([[ id:'test_query', single_end:true ],input_file] ).untar
+    ch_target = UNTAR_TARGET( [[ id:'test_target', single_end:true ],input_file] ).untar
+    ch_result = UNTAR_RESULT( [[ id:'test_result', single_end:true ],input_file] ).untar
```
Comment (JoseEspinosa) [inline tests/modules/nf-core/mmseqs/createtsv/main.nf:17]:
```suggestion
    ch_query = UNTAR_QUERY ([[ id:'test_query', single_end:true ], input_file] ).untar
    ch_target = UNTAR_TARGET( [[ id:'test_target', single_end:true ], input_file] ).untar
    ch_result = UNTAR_RESULT( [[ id:'test_result', single_end:true ], input_file] ).untar
```

File: tests/modules/nf-core/mmseqs/createtsv/main.nf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    MMSEQS_CREATETSV (ch_result, ch_query, ch_target  )
+}
+
+workflow test_mmseqs_createtsv_optional {
+
+    input_file = file(params.test_data['sarscov2']['genome']['mmseqs_tar_gz'], checkIfExists: true)
+
+    ch_query = UNTAR_QUERY2 ([[ id:'test_query', single_end:true ],input_file] ).untar
+    ch_result = UNTAR_RESULT2( [[ id:'test_result', single_end:true ],input_file] ).untar
```
Comment (JoseEspinosa) [inline tests/modules/nf-core/mmseqs/createtsv/main.nf:27]:
```suggestion
    ch_query = UNTAR_QUERY2 ([[ id:'test_query', single_end:true ], input_file] ).untar
    ch_result = UNTAR_RESULT2( [[ id:'test_result', single_end:true ], input_file] ).untar
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
LGTM! Just some small format suggestions

**Pull request merged**